### PR TITLE
docs: add Week 5 general announcement

### LINKED ISSUE #34: Week 5 Announcement Brain Dump
Congrats on the excellent feedback about Task 2.1
Starting to think about your retrospective (6.1) / task 5.1
Watch the D/HD workshop video (benefitial for everyone)
Fill in the learning survey
Seniors who want to mentor – reach out
Juniors who would like to work towards leadership next trimester (EOI for attending the leadership conference being organised)
Showcase for D/HD students being organised at the end of the week - presenting to MDs with trello board, plans, testing & data strategies (fill in the templates) - great place for feedback
By now trello boards should planned out, upskilling should already be happening
Comfortable in github, creating pull requests
Review the QA doc – there will be a session on this during the week
Transition away from word docs – everything moving towards thoth tech from this week onward.
Ensure you have cloned your repos from the Thoth Tech organisation - not doubtfirelms

### PR DESCRIPTION
**Pull request by tancnle**
# Description

Add Week 5 General Announcement

Fixes #34

## Type of change

- [x] Documentation (update or new)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Good Evening Thoth Tech.
+
+Here is this week's update from your friendly neighbourhood leadership team.
+
+First and foremost, congratulations on the excellent feedback on Task 2.1! It
+was a team effort and everyone deserves a pat on the back.
+
+## OnTrack tasks
+
+For D/HD aiming students, you should start working on Task 5.1 retrospective.
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Task 5.1 isn't a retrospective - might get a bit confusing

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- Please review the Quality Assurance (QA)
+  [document](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/quality-assurance-overview.md)
+  and start a conversation around the topic with your team. There will be a
+  session on this during the week (Wed 6/04/2022 6:00 PM - 6:45 PM) and it is
+  open to everyone.
+- Please review the Communication
+  [Guide](https://github.com/thoth-tech/handbook/blob/main/docs/communication/communication.md)
+  and see if it can help to streamline your team communication process.
+- Any seniors who want to mentor, please reach out to the Leadership team.
+- Any juniors who would like to work towards leadership next trimester. We are
```
Comment (kats8) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]:
```suggestion
- Any juniors who would like to work towards leadership next trimester, we are
```

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  organising an Expression of Interest for attending the leadership conference.
+  Please stay tuned for more information.
+- Please work on the relevant project repositories from **the Thoth Tech
+  organisation**. Please do not pull or push to Doubtfire or SplashKit
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Maybe add in "the original doubtfire or Splashkit" as they may get confused

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  repositories.
+
+## What you can expect
+
+- Showcase for D/HD students is being organised at the end of the week. We would
+  like to present project Trello boards, plans, testing & data strategies to the
+  Managing Directors. It would be a great opportunity to receive feedback.
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md:48]: Please speak to your delivery leads to confirm asap

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+## What we can expect
+
+- We expect that Trello boards should be planned out and upskilling should
+  already be happening. If you need help, please reach out
+- Team members should feel comfortable with GitHub and creating pull requests.
+  Please review the training
+  [video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Recordings/24-3-22%20Github%20Repos%20QnA.mp4?csf=1&web=1&e=Pa3Hh0)
+  and the QA
+  [document](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/quality-assurance-overview.md)
+  if you are still unsure.
```
Comment (ginge270) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: If you have done both of these things and still feel lost - please reach out to the area leads

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+- Please work on the relevant project repositories from **the Thoth Tech
+  organisation**. Please do not pull or push to Doubtfire or SplashKit
+  repositories.
+
+## What you can expect
+
+- Showcase for D/HD students is being organised at the end of the week. We would
+  like to present project Trello boards, plans, testing & data strategies to the
+  Managing Directors. It would be a great opportunity to receive feedback.
+- We are making transition away from MS Word docs to Thoth-tech
```
Comment (kats8) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]:
```suggestion
- We are making a transition away from MS Word docs to the Thoth-tech
```

(or 'We are transitioning away')

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 5 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Addressed in e6c530b

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 4 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md:48]: Addressed in e6c530b

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 3 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Addressed in e6c530b

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 2 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Addressed in 7e5c9285002f

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 6 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Addressed in [88e2bd0](https://github.com/thoth-tech/handbook/pull/48/commits/88e2bd04ca0081c82c126d66d3d9677d8cdb5a37)

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md (hunk 1 above)
Comment (tancnle) [reply docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md]: Addressed in [88e2bd0](https://github.com/thoth-tech/handbook/pull/48/commits/88e2bd04ca0081c82c126d66d3d9677d8cdb5a37)

**Comment by tancnle** [comment]
Thank you for the detailed review @ginge270 and @kats8 ❤️. I have incorporated all of your suggestions. Please let me know what you think 🙏🏼

**Comment by tancnle** [comment]
Any other thoughts @ginge270 @kats8 @talzid or can I start circulating it in Teams now?

**Review by ginge270 — APPROVED** [review]

**Pull request merged**